src/pages/login/login.ts

```
@@ -35,15 +35,22 @@ export class LoginPage {
     ).then(
       () => this.info.init()
     ).then(
-      () => this.navCtrl.setRoot(IncidentsPage)
+      () => {
+              this.app.getRootNav().setRoot(TabsPage);
+              this.navCtrl.setRoot(TabsPage);        
+      }
+  
     )
```

Errors are not handled here and possibly swallowed.